pkgs/development/libraries/qt-5/5.9/qtbase/default.nix

```diff
@@ -114,6 +115,10 @@ stdenv.mkDerivation {
               --replace "/System/Library/Frameworks/OpenGL.framework/" "${darwin.apple_sdk.frameworks.OpenGL}/Library/Frameworks/OpenGL.framework/"
           substituteInPlace ./mkspecs/common/mac.conf \
               --replace "/System/Library/Frameworks/AGL.framework/" "${darwin.apple_sdk.frameworks.AGL}/Library/Frameworks/AGL.framework/"
+          substituteInPlace ./mkspecs/common/mac.conf \
+              --replace "-framework OpenGL" "${darwin.apple_sdk.frameworks.OpenGL}/Library/Frameworks/OpenGL.framework/OpenGL"
```

Comment:
this looks a little strange, `-framework Foo` should work correctly when it's added as a dependency.

Reply:
Removed. I ran into some linking issue when build an app depends on qt. I guess the app itself need some change